tests/test_mock_devices.py

```diff
@@ -394,12 +394,12 @@ async def check_axis_device(self, device):
         await asyncio.sleep(0.1)  # give command time to start
         self.assertAlmostEqual(device.actuator.target.position, end_position)
         self.assertEqual(device.actuator.target.velocity, 0)
```

Comment:
Perhaps replace this with `assert`s as well? And please check the rest of the code in this file as well.